json/flags.yaml

```diff
@@ -99,20 +99,20 @@ properties:
         patternProperties:
           "^.{1,}$":
             type: object
-  $comment: merge the variants with the base flag to build our typed flags
   booleanFlag:
+    $comment: merge the variants with the base flag to build our typed flags
```

Comment:
This is the only unrelated change... for some reason changing `$defs` to `definitions` resulted in saying the location of this commend was previously invalid.